ui-spacetimechart/src/components/workSchedules/Region.tsx

```diff
@@ -0,0 +1,57 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import { useDraw } from '../../hooks/useCanvas';
+import { type DrawingFunction } from '../../lib/types';
+import { type SpaceTimeRegion } from '../../types';
+
+const Region = ({ timeStart, timeEnd, spaceStart, spaceEnd, image }: SpaceTimeRegion) => {
```

Comment:
I don't really like the term Region, because it's not only a space object, but a space object during a certain period of time. 
SpaceTimeRectangle might be a better name

Reply:
SpaceTimeRectangle it is :)

Reply:
Hm, this doesn't reflect the fact that it takes an image and repeats it in a pattern.

Reply:
As it’s in the spacetime chart folder, Seeing this now I tend to agree more with PatternRect

Reply:
Fine for me :+1: